src/main/java/DeadlineCommand.java

```diff
@@ -0,0 +1,11 @@
+/**
+ * Represents a "deadline" command give to TaskList.
+ */
+public class DeadlineCommand extends Command {
+
+    public DeadlineCommand(Deadline d) {
+        super();
+        this.command = "deadline";
```

Comment:
Instead of hard-coding the strings, you can create `enums` ints for the command instead (and storing the command codes as `int` instead of `String`)